src/app/shared/ui/notification-message/models/notification-config.model.ts

```diff
@@ -0,0 +1,12 @@
+export const NOTIFICATION_TYPE_CONSTANTS = {
+    SUCCESS: 'SUCCESS',
+    WARNING: 'WARNING',
+    ERROR: 'ERROR'
+} as const
+
+export type NOTIFICATION_TYPE = typeof NOTIFICATION_TYPE_CONSTANTS[keyof typeof NOTIFICATION_TYPE_CONSTANTS]
```

Comment:
Instead of using enums, we can use constants, but requires this line of code.
Currently, we will need the constant object to reference in template 😆 
https://dev.to/maafaishal/you-might-not-need-typescript-enum-1f4n